pages/demo/test.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+  <div class="block h-screen w-full">
+    <input ref="place" v-model="value" class="h-5 w-1/2 bg-gray-200" />
+    <button class="bg-blue-500" @click="save">{{ title }}</button>
+    <div id="map" class="mt-5 h-1/2 w-1/2 bg-gray-500"></div>
+  </div>
+</template>
+
+<script>
+import firebase from 'firebase/app'
+import 'firebase/firestore'
+import { getGoogle } from '~/use/google'
+
+export default {
+  data: () => ({
+    value: '',
+    result: '',
+    title: 'Save',
+  }),
+  methods: {
+    save() {
+      this.getDateFromText(this.value)
```

Comment:
if then, this function has to be changed with `this.getVenueFromText(this.value)`